### PR TITLE
Implement custom popup for Smartspacer

### PR DESCRIPTION
**Pull request by suphon-t**
## Description

Replaced Smartspacer's popup with Launcher3's popup, and adda shortcut to Smartspacer settings in Smartspace preferences.

## Type of change

:white_check_mark: General change (non-breaking change that doesn't fit the below categories like copyediting)
:x: Bug fix (non-breaking change which fixes an issue)
:x: New feature (non-breaking change which adds functionality)
:x: Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by KieronQuinn** [comment]
This looks fine, but it has actually removed functionality. The popup menu dynamically shows more than just customise, it also has options to dismiss and open feedback/about intents (same as Smartspace in the Pixel Launcher). The default popup also implements these, so you can copy that to see what they're meant to do: https://github.com/KieronQuinn/Smartspacer/blob/main/sdk-client/src/main/java/com/kieronquinn/app/smartspacer/sdk/client/views/popup/BalloonPopupFactory.kt

As you've done the actual work implementing this, I can probably pick it up from here and implement the rest if you'd prefer?

**Comment by suphon-t** [comment]
> As you've done the actual work implementing this, I can probably pick it up from here and implement the rest if you'd prefer?

Yeah, sure. That would be helpful since you know more about the UX. Thanks a lot.
Let's do a PR to this branch, and then merge everything together.

**Comment by KieronQuinn** [comment]
> > As you've done the actual work implementing this, I can probably pick it up from here and implement the rest if you'd prefer?
> 
> Yeah, sure. That would be helpful since you know more about the UX. Thanks a lot. Let's do a PR to this branch, and then merge everything together.

Done in #3725

**Review by MrSluffy — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -172,8 +172,8 @@ android {
         debug {
             applicationIdSuffix ".debug"
             resValue("string", "derived_app_name", "Lawnchair (Debug)")
-            minifyEnabled true
-            shrinkResources true
+            minifyEnabled false
+            shrinkResources false
```
Comment (Goooler) [inline build.gradle:176]: Why tunning off minify here?

**Pull request merged**